lib/games_collection.rb

```diff
@@ -14,4 +14,32 @@ def initialize(games_path)
   def create_games(games_path)
     create_instances(games_path, Game)
   end
+
+  def average_goals_per_game
+    goals_per_game = @games.map {|game| game.away_goals + game.home_goals}
+    all_goals = goals_per_game.sum
```

Comment:
Look at combining these to into a .sum.  IE: @games.sum  { |game| game.away_goals + game.home_goals }